src/partials/hero.html

```diff
@@ -1,2 +1,39 @@
-<section class="hero">
-</section>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GreenHarvest</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter+Tight:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/2.0.0/modern-normalize.min.css" integrity="sha512-4xo8blKMVCiXpTaLzQSLSw3KFOVPWhm/TRtuPVc4WG6kUgjH6J03IBuG7JZPkcWMxJ5huwaBpOpnwYElP/m6wg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="../css/hero.css" />
+  
+</head>
+<body>
+    <section class="hero">
+        <div class="hero-wrap container">
+            <h1 class="hero-title">organic vegetables to your diet today!</h1>
+            <p class="hero-text">Our expertly curated vegetable baskets are made with the freshest, highest quality vegetables available. Fresh Harvest Box has got you covered. Fresh, high-quality vegetables in expertly curated vegetable baskets delivered to you.</p>
+        </div>
+        <picture class="hero-img">
+            <source
+            srcset="../img/hero-img/hero1x-desk.jpg 1x, ../img/hero-img/hero2x-desk.jpg 2x"
+              media="(min-width: 1280px)"
+            />
+            <source
+              srcset="../img/hero-img/hero1x-tab.jpg 1x, ../img/hero-img/hero2x-tab.jpg 2x"
+              media="(min-width: 768px)"
+            />
+            <source
+              srcset="../img/hero-img/hero1x-mob.jpg 1x, 
+              ../img/hero-img/hero2x-mob.jpg 2x"
+              media="(min-width: 375px)"
+            />
+            <img src="../img/hero-img/hero1x-mob.jpg" alt="hero-img">
+        </picture>
+        </section>
```

Comment:
line 37, closing tag section cant be on the same position with other closing tag.